src/app/public/index.html

```diff
@@ -71,9 +71,45 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
     <script type="text/javascript">
-      function googleTranslateElementInit() {
-        new google.translate.TranslateElement({pageLanguage: 'en', layout: google.translate.TranslateElement.InlineLayout.SIMPLE, autoDisplay: true}, 'google_translate_element');
-      }
+        function googleTranslateElementInit() {
+            var translateElementID = 'google_translate_element';
+
+            var translateElement = new google.translate.TranslateElement({
+                pageLanguage: 'en',
+                layout: google.translate.TranslateElement.InlineLayout.SIMPLE,
+                autoDisplay: true,
+            }, translateElementID);
+
+            var callback = function () {
+                var el = document.getElementsByClassName('goog-te-menu-value');
+
+                if (el.length) {
+                    var navbarElement = el[0];
+                    var childNodesLength = navbarElement.childNodes.length;
```

Comment:
Assigned out of the loop because otherwise the .length property would decrement as nodes were removed.